### PR TITLE
Added PHP linter.

### PR DESCRIPTION
**Pull request by jgmdev**

### CODE REVIEW

**Comment by liquidev** [comment]
I'm not familiar with PHP, I suppose the interpreter doesn't ever emit any warnings?

Looks good otherwise.

**Comment by jgmdev** [comment]
> I suppose the interpreter doesn't ever emit any warnings (only errors)?

Right, the php built-in linter only checks for syntax errors, every other error is catch at runtime. There are static code analyzers like [phan](https://github.com/phan/phan), [phpstan](https://phpstan.org/) and [psalm](https://psalm.dev/) that do report different kind of error levels but I guess a separate linter plugin would be needed to be written for each of those that can analyze entire code bases and run as a background service.

**Comment by liquidev** [comment]
Yeah, that's fine with me. If anyone needs some extra linting functionality, they can create a separate, more advanced linter that uses one of the analyzers you linked.

File: lintplus_php.lua [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+--   passes the specified arguments to php
+
+--- IMPLEMENTATION ---
+
+local lintplus = require "plugins.lintplus"
+
+local redirection = ""
+if PLATFORM == "Linux" then
+  redirection = "2> /dev/null"
+end
```
Comment (liquidev) [inline lintplus_php.lua:17]: I just noticed this while reviewing the code a second time. I'm a afraid that redirection thingy isn't going to work properly when lintplus is in async mode, as then the arguments are not concatenated into a shell command, but rather run as a process directly via a native library.

Is there any extra output PHP has that goes into stderr? If so, does it match the pattern string you specified in the interpreter?

Comment (jgmdev) [reply lintplus_php.lua]: Removed the redirection, the purpose was to reduce the amount of output that lite shows on the terminal.

**Pull request merged**